### PR TITLE
Ghost Cafe: Themed Rooms. (also paperbins) [ready]

### PR DESCRIPTION
**Pull request by thestubborn**
## About The Pull Request

Turns out the old ghouls that run the Cafe are alive! They've had their trans-dimensional workers upgrade the Cafe with the most advanced technology the galaxy knows!
Paper bins.
Oh, they also made some themed rooms or something.
image
The Hive is a horror-themed room with authentically fake Xenomorph beds! (Includes a free toy facehugger!)
image
The Mothership is a scifi-themed room with cold metal beds and an authentic zetan styled locker!

image

Management has also moved some vendors around, rumours that they just threw the old sec vendor into a cave are completely false.

image
The Lib got some paper, painting frames (and materials, added that after the screen)

image
Janidrobe was moved to the broom closet.

image
Some deadbeat had their musical gear re-acquired.

Oh, and every bed now has a random sheet instead of just blue.

## How This Contributes To The Skyrat Roleplay Experience

flavour good
themed rooms give cafe people a reason to not just go into the hotel ball, it also gives way for more out there roleplay experiences you cant, or can only rarely get elsewhere

aka xenofucking

## Changelog

:cl:
add: Ghost Cafe management are NOT dead! They have just added some brand new themed rooms (and the other goodies...), go check them out!
/:cl:

### CODE REVIEW

**Comment by thestubborn** [comment]
and before someone shouts at me about some ghost cafe thing they were doing

I cant see discord rn, still chatbanned

and if it's a minor fix I'll throw it in

**Comment by thestubborn** [comment]
image
oh yeah, the updated lib

**Comment by thestubborn** [comment]
image
lab looking more alien

**Comment by GoldenAlpharex** [comment]
Also, if you're making changes to the Ghost Cafe, I think we should move towards using double beds there, rather than simple ones, because they're just cooler and more 2022. 😎

File: _maps/map_files/generic/CentCom_skyrat_z2.dmm [hunk 1]
```diff
@@ -6253,12 +6308,39 @@
...
+/obj/effect/turf_decal/tile/neutral{
+	dir = 8
+	},
+/obj/effect/turf_decal/trimline/blue/filled/line,
+/obj/machinery/vending/autodrobe{
+	default_price = 0;
+	extended_inventory = 1;
+	extra_price = 0;
+	fair_market_price = 0
+	},
```
Comment (GoldenAlpharex) [inline _maps/map_files/generic/CentCom_skyrat_z2.dmm]: I don't actually think you need any of that, it loads at round-start and it's off-station, so it should be free.

File: _maps/map_files/generic/CentCom_skyrat_z2.dmm [hunk 2]
```diff
@@ -7993,6 +8154,15 @@
 /obj/structure/flora/ausbushes/sparsegrass,
 /turf/open/floor/plating/grass/planet,
 /area/centcom/interlink)
+"rne" = (
+/obj/item/paper{
+	info = "Hello valued Patron! The Cafe has underwent recent renovations! The most noticable additions are a horror themed 'Hive' room in the southeast (please note that due to this theme, there are currently no doors, managment is looking into adding them) and a Sci-Fi 'Alien abduction' theme, just south of the Drobe room. Thank you, -The Managment.";
```
Comment (GoldenAlpharex) [inline _maps/map_files/generic/CentCom_skyrat_z2.dmm]:
```suggestion
	info = "Hello valued Patron! The Cafe has underwent recent renovations! The most noticable additions are a horror themed 'Hive' room in the southeast (please note that due to this theme, there are currently no doors, managment is looking into adding them) and a Sci-Fi 'Alien abduction' theme, just south of the Drobe room. Thank you,\n -The Managment.";
```

File: _maps/map_files/generic/CentCom_skyrat_z2.dmm [hunk 3]
```diff
@@ -8577,6 +8794,31 @@
...
+/obj/item/instrument/saxophone,
+/obj/item/instrument/trombone,
+/obj/item/instrument/trumpet,
+/obj/item/instrument/violin,
+/obj/item/reagent_containers/food/condiment/mayonnaise,
+/obj/structure/closet/crate/wooden{
+	name = "band crate"
+	},
+/obj/item/paper{
+	info = "Dear Mr. Best, due to your overwhelming and unpaid tab, Managment has saw fit to confiscate your equipment, we do not care which mammel themed band you claim to be a member of. Thank you. -Managment.";
```
Comment (GoldenAlpharex) [inline _maps/map_files/generic/CentCom_skyrat_z2.dmm]:
```suggestion
	info = "Dear Mr. Best, due to your overwhelming and unpaid tab, Managment has saw fit to confiscate your equipment, we do not care which mammel themed band you claim to be a member of.\nThank you.\n-Managment.";
```

**Review by GoldenAlpharex — CHANGES_REQUESTED** [review]
We're missing quite a few walls in the new room in the ayyy room.

As I mentioned earlier, most (if not all) best should be made into double beds, with the proper double bedsheet spawner too.

I'm not quite sure why there's an old note and a knife on the vendor, it's not really needed, in my opinion. It'd be relevant for a whooping like week, and then would be irrelevant afterwards.

Got a few more comments down there too.

File: _maps/map_files/generic/CentCom_skyrat_z2.dmm (hunk 1 above)
Comment (thestubborn) [reply _maps/map_files/generic/CentCom_skyrat_z2.dmm]: its a holdover from like, old days

**Comment by thestubborn** [comment]
> We're missing quite a few walls in the new room in the ayyy room.
> 
> As I mentioned earlier, most (if not all) best should be made into double beds, with the proper double bedsheet spawner too.
> 
> I'm not quite sure why there's an old note and a knife on the vendor, it's not really needed, in my opinion. It'd be relevant for a whooping like week, and then would be irrelevant afterwards.
> 
> Got a few more comments down there too.

the alien corner thing is intentional, as the corners bend, meaning that it looks weird next to the door
the note I can remove, I added it so if people wanted redsec they have a hint of where it is
I can do doublebeds, yeah

**Comment by thestubborn** [comment]
unless you'd like me to make the *entire* lab an alien location?

File: _maps/map_files/generic/CentCom_skyrat_z2.dmm (hunk 1 above)
Comment (thestubborn) [reply _maps/map_files/generic/CentCom_skyrat_z2.dmm]: but removed anyway, yeah

**Comment by thestubborn** [comment]
image
this is what i mean with the corners

**Comment by thestubborn** [comment]
image
made the entire lab alien, i'll screenshot ig rn

**Comment by thestubborn** [comment]
image

**Comment by thestubborn** [comment]
those are all fake, by the way
simple icon swaps

**Review by KathrinBailey — CHANGES_REQUESTED** [review]
You can gain one free MergePass™ exclusively for this PR only by reverting the changes that appear outside the ghost cafe on mapdiffbot's differences.

**Comment by thestubborn** [comment]
oh fair, i was meanin to do that in a dff pr anyway

**Comment by thestubborn** [comment]
> You can gain one free MergePass™ exclusively for this PR only by reverting the changes that appear outside the ghost cafe on mapdiffbot's differences.

right I think I reverted it

**Review by KathrinBailey — APPROVED** [review]
image

**Comment by thestubborn** [comment]
> image

image

**Pull request merged**